sdks/python/apache_beam/internal/metrics/metric.py

```diff
@@ -223,3 +223,31 @@ def convert_to_canonical_status_string(self, status):
         http_status_code in http_to_canonical_gcp_status):
       return http_to_canonical_gcp_status[http_status_code]
     return str(http_status_code)
+
+  @staticmethod
+  def convert_to_grpc_status_string(grpc_status):
```

Comment:
please rename variable grpc_status -> grpc_status_code

Reply:
please rename function to:
bigtable_error_code_to_grpc_status_string

Reply:
Done